changelog_unreleased/mdx/pr-6398.md

```diff
@@ -0,0 +1,24 @@
+#### MDX: fix JSX Fragment format incorrectly ([#6398] by [@JounQin])
```

Comment:
We don't write links like this anymore in the changelog. Please change to this syntax: `[...](https://...)`

Reply:
Done.